manual/src/html_processing/README.md

```diff
@@ -15,6 +15,21 @@ There are two different scripts, `process_manual.ml` and
 manual, while the latter deals with the api generated with `ocamldoc`.
 They both use a common module `common.ml`.
 
+## Prerequisites
+
+In order to do postprocessing of the HTML manual, ensure that you have
+the `sass` package installed:
+
+```
+apt install sass
```

Comment:
It may be useful to add that,

"ensure that you have the `sass` package installed. On Debian-based Linux distributions, you can install that package by:"